bin/configs/jaxrs-spec-quarkus-mutiny.yaml

```diff
@@ -0,0 +1,15 @@
+generatorName: jaxrs-spec
+outputDir: samples/server/petstore/jaxrs-spec-quarkus-mutiny
```

Comment:
can you please add the new output folder to `.github/workflows/samples-jaxrs.yaml` to that the CI will test it moving forward?